DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -3385,49 +3385,49 @@ defaultproperties
     bingo_options(213)=(event="BethsPainting",desc="Admire Beth DuClare's favourite painting",max=1,missions=1024)
 #endif
 #ifndef hx
-    bingo_options(214)=(event="ViewPortraits",desc="Look at %s portraits",max=2,missions=4890)
+    bingo_options(214)=(event="ViewPortraits",desc="Look at %s portraits",desc_singular="Look at 1 portrait",max=2,missions=4890)
     bingo_options(215)=(event="ViewSchematics",desc="Inspect a schematic",max=1,missions=49152)
-    bingo_options(216)=(event="ViewMaps",desc="View %s maps",max=6,missions=56686)
+    bingo_options(216)=(event="ViewMaps",desc="View %s maps",desc_singular="View 1 map",max=6,missions=56686)
     bingo_options(217)=(event="ViewDissection",desc="Have a look at a dissection report",max=1,missions=96)
     bingo_options(218)=(event="ViewTouristPics",desc="Look at a tourist picture",max=1,missions=2576)
 #endif
     bingo_options(219)=(event="CathedralUnderwater",desc="Swim through the underwater tunnel at the cathedral",max=1,missions=2048)
     bingo_options(220)=(event="DL_gold_found_Played",desc="Recover the Templar gold",max=1,missions=2048)
     bingo_options(221)=(event="12_Email04",desc="Read a motivational email from Gary",max=1,missions=4096)
-    bingo_options(222)=(event="ReadJCEmail",desc="Check your email %s times",max=3,missions=122)
+    bingo_options(222)=(event="ReadJCEmail",desc="Check your email %s times",desc_singular="Check your email 1 time",max=3,missions=122)
     bingo_options(223)=(event="02_Email05",desc="Paul's Classic Movies",max=1,missions=4)
     bingo_options(224)=(event="11_Book08",desc="Read Adept 34501's diary",max=1,missions=2048)
     bingo_options(225)=(event="GasStationCeiling",desc="Access the ceiling of a gas station",max=1,missions=4096)
-    bingo_options(226)=(event="NicoletteHouseTour",desc="Tour %s parts of Chateau DuClare with Nicolette",max=5,missions=1024)
+    bingo_options(226)=(event="NicoletteHouseTour",desc="Tour %s parts of Chateau DuClare with Nicolette",desc_singular="Tour 1 part of Chateau DuClare with Nicolette",max=5,missions=1024)
     bingo_options(227)=(event="nico_fireplace",desc="Access Nicolette's secret stash",max=1,missions=1024)
     bingo_options(228)=(event="dumbwaiter",desc="Not so dumb now!",max=1,missions=1024)
     bingo_options(229)=(event="secretdoor01",desc="Open the secret door in the cathedral",max=1,missions=2048)
     bingo_options(230)=(event="CathedralLibrary",desc="Worth its weight in gold",max=1,missions=2048)
-    bingo_options(231)=(event="DuClareKeys",desc="Collect %s different keys around Chateau DuClare",max=3,missions=1024)
-    bingo_options(232)=(event="ShipLockerKeys",desc="Collect %s locker keys inside the superfreighter",max=2,missions=512)
+    bingo_options(231)=(event="DuClareKeys",desc="Collect %s different keys around Chateau DuClare",desc_singular="Collect 1 key in Chateau DuClare",max=3,missions=1024)
```

Comment:
should this one vary or always be 3?

Reply:
They're rather outside the original scope of this PR but I agree with these changes.

Reply:
I'll do it after merging this